### PR TITLE
austin: Add version 1.0.1

### PR DESCRIPTION
**Pull request by P403n1x87**
Austin is a Python frame stack sampler for CPython

### CODE REVIEW

File: bucket/austin.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        "url": "https://github.com/P403n1x87/austin/blob/master/LICENSE.md"
+    },
+    "version": "1.0.1",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/P403n1x87/austin/releases/download/v1.0.1/austin-1.0.1-win64.msi",
+            "hash": "74864412DF9A44760B9CEBDF4D00706FA8795FA3E5F3DD290AF041C37922A50D"
+        }
+    },
+    "extract_dir": "austin",
```
Comment (Ash258) [inline bucket/austin.json:15]:
```suggestion
    "version": "1.0.1",
    "description": "Python frame stack sampler for CPython",
    "homepage": "https://github.com/P403n1x87/austin",
    "license": "GPL-3.0-or-later",
    "architecture": {
        "64bit": {
            "url": "https://github.com/P403n1x87/austin/releases/download/v1.0.1/austin-1.0.1-win-x86_64.zip",
            "hash": "9b3f59383cfbf633d54a01b72a42618a9a70a31444997fcdb8bc2fce2cd8c80b"
        }
    },
```

File: bucket/austin.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "bin": "austin.exe",
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/P403n1x87/austin/releases/download/v$version/austin-$version-win64.msi"
```
Comment (Ash258) [inline bucket/austin.json:21]:
```suggestion
                "url": "https://github.com/P403n1x87/austin/releases/download/v$version/austin-$version-win-x86_64.zip"
```

**Comment by P403n1x87** [comment]
> Make no sense to use msi when there is zip provided.
> 
> Why did you choose msi?
> 
> Is python installation required?

Hi. Thanks for the feedback. I thought msi would have been the cleaner approach, but given that I now understand that the msi is unzipped rather than installed anyway, of course the ZIP is the best option :). Python is, in principle, not required in order to run Austin. Of course, since Austin is a Python profiler, you'd need a Python application to profile.

**Pull request merged**